### DISCUSSION TITLE
Issues running under Raspbian Bullseye

### DISCUSSION THREAD

**Discussion by GaelNguyen** [CoolerMasterTechnology/Pi-Tool#20] [General]
I just loaded the latest version of Raspbian and the Pi Tool seems to install without a problem however when I try to open the app it won't run. Has anyone else found issues with the latest versions of Raspbian?

**Comment by kodlz** [comment]
HI,
I have similar issue,
I just try to install it on new Raspberry OS 11 arm64.
It looks like one of lib is not available for Bullseye

> pi-tool : Depends: **libappindicator1** but it is not installable

probably that is reason why is not possible run Pi-tool.

log from installation is attached
[pi-tool_install.log](https://github.com/CoolerMasterTechnology/Pi-Tool/files/7611909/pi-tool_install.log)
